src/main/java/com/kammatysiak/medicalclinic/controller/PatientController.java

```diff
@@ -0,0 +1,53 @@
+package com.kammatysiak.medicalclinic.controller;
+
+import com.kammatysiak.medicalclinic.model.dto.PasswordClassDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientCreateDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientDTO;
+import com.kammatysiak.medicalclinic.model.entity.Patient;
+import com.kammatysiak.medicalclinic.service.PatientService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/patients")
+public class PatientController {
+
+    private final PatientService patientService;
+//w controllerze definiujemy endpointy (punkty komunikacji) a serwis ma dostarczyc wlasciwe dane do kontrolera
+
+    @GetMapping
+    public List<PatientDTO> getPatients() {
+        return patientService.getPatients();
+    }
+
+    @GetMapping("/{email}")
+    public PatientDTO getPatient(@PathVariable String email) {
+        return patientService.getPatient(email);
+    }
+
+    @PostMapping
+    public PatientDTO postPatient(@RequestBody PatientCreateDTO patientDTO) {
+       return patientService.createPatient(patientDTO);
+    }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Status nie potrzebny jeśli używam wyjątków z ControllerAdvice

Reply:
to jest status sukcesu, ustalany ze wzgledu na to, ze metoda ta ma zwracac nic (void)

Reply:
hehe xd